build/Targets/VSL.Versions.targets

```diff
@@ -82,6 +82,16 @@
     </Otherwise>
   </Choose>
 
+  <PropertyGroup>
+    <GitHeadSha>Not found</GitHeadSha>
```

Comment:
Yeah, I explicitly don't want this for just official builds. If I get some random unsigned build I also want to know if it came from a known SHA.